doc/source/tune.rst

```diff
@@ -67,49 +67,7 @@ If using TF2 and TensorBoard, Tune will also automatically generate TensorBoard
     :scale: 20%
     :align: center
 
-Distributed Quick Start
```

Comment:
Is this content moved somewhere else or perhaps no longer useful?

Reply:
No longer useful; I figured our local mode autoscaler is not really usable and the other content on the page can have a higher impact if surfaced first.